libraries/lib-sample-track/MixerSource.cpp

```diff
@@ -70,7 +70,7 @@ size_t MixerSource::MixVariableRates(
    unsigned iChannel, const size_t maxOut, float &floatBuffer)
 {
    auto &cache = mInputSequence[iChannel];
-   const auto pos = &mSamplePos[iChannel];
+   auto pos = mSamplePos[iChannel];
```

Comment:
Am I correct in understanding that this commit doesn't change any behaviour?

Reply:
Correct, the commit "Update mSamplePos in just two places, at function exits" is not intended to change behavior.

The transformation of `MixerSource` needs to be careful, and I'm showing small steps of transformation in these commits.  And still I made a mistake which now has its fix-up commit.